_docs/extend/improvements.md

```diff
@@ -5,7 +5,7 @@ description: Here's a list of improvements that can be made to Glacier, the
 ---
 
 
-## Don't store electronic copy of Cold Storage Information Page
+## Don't store electronic copy of cold storage information page
```

Comment:
Cold Storage Information Page is capitalized as a proper noun throughout the doc; shouldn't it be capitalized here too? (Like you left Shamir's Secret Sharing capitalized)

Reply:
Corrected in a new commit.